roles/edpm_nodes_validation/meta/argument_specs.yml

```diff
@@ -16,10 +16,18 @@ argument_specs:
         description: Attempt to reach controllers with ping.
         type: bool
         default: true
+      edpm_nodes_validation_check_for_fqdn:
+        description: Verify if an FQDN is set. Typically required for TLS-e.
```

Comment:
Shouldn't imply this is only required for TLS-e (it really cares) since it will silently break lots of other stuff too